.gitignore

```diff
@@ -1,3 +1,4 @@
 bedrock-linux-*.sh
 build
 vendor
+.*.swp
```

Comment:
I prefer not to add non-specific-codebase-related things like this to `.gitignore`.  Happily, git provides alternatives which you may find suitable.

For user-specific things which are also repo-specific - if you only generate `*~` files when working on Bedrock - git offers `$GIT_DIR/info/exclude`.  From `man 5 gitignore`:

> Patterns which are specific to a particular repository but which do not need to be shared with other related repositories (e.g., auxiliary files that live inside the repository but are specific to one user’s workflow) should go into the $GIT_DIR/info/exclude file.

If you create `*~` files all over the place and you want git to ignore them everywhere, you can use `core.excludeFile` (which defaults to `~/.config/git/ignore`):

> Patterns which a user wants Git to ignore in all situations (e.g., backup or temporary files generated by the user’s editor of choice) generally go into a file specified by core.excludesFile in the user’s ~/.gitconfig. Its default value is $XDG_CONFIG_HOME/git/ignore. If $XDG_CONFIG_HOME is either not set or empty, $HOME/.config/git/ignore is used instead.

Reply:
Finished, altough you copied this comment from Exherbo PR and wrote about `*~` instead of `.*.swp`

Reply:
That I did >.<